front/src/Api/IframeListener.ts

```diff
@@ -143,65 +148,101 @@ class IframeListener {
 
                 foundSrc = this.getBaseUrl(foundSrc, message.source);
 
-                if (isIframeEventWrapper(payload)) {
+                if (isIframeQueryWrapper(payload)) {
```

Comment:
maybe a bit uncertain if theits actually a queryWrapper  id  + type string is quite common

Reply:
The structure of the message the type guard is checking is:

```
{ id: string,
  query: {
    type: string  
  }
}
```

So it's not that common. I think we are safe here.